.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 # NOTES:
```

Comment:
this comments blocks merge and should only be resolved once pre `0.8.0` stuff has been released

Reply:
and rebase necessary anyway ...

Reply:
still safe so no one gets an idea an reabses+merges this too early :O